src/components/PlaceholderContainer/README.md

```diff
@@ -4,19 +4,19 @@
 
 <!--/GITHUB_BLOCK-->
 
-`PlaceholderContainer` is a component for displaying content with image, text content and action controls.
+`PlaceholderContainer` is a component for displaying content with images, text content, and action controls.
```

Comment:
Why images in plural?

Reply:
Got a comment from loc team: "cause if you use a countable noun, you either use an article or a plural form. I was using my best guess here. if there may only be one image, use an image."

Changed to "an image".